DFC.EventGridSubscriptions.ApiFunction/Function/Execute.cs

```diff
@@ -120,16 +129,34 @@ private bool ValidateBodyParameters(SubscriptionRequest request, out string mess
                 return false;
             }
 
+            if (request.Name.Length <= 3 || request.Name.Length > 64 || Regex.Match(request.Name, "^[a-zA-Z 0-9\\-]*$").Captures.Count == 0)
```

Comment:
Could you just have a regex and check the length with that, using something like {3,64}?

Reply:
Yeah I could whack it all into one I just did the Regex after, I'll consolidate :)